static/js/search.js

```diff
@@ -180,6 +180,14 @@ exports.initialize = function () {
         // short enough that the user won't notice (though
         // really it would be OK if they did).
 
+        if ($(e.relatedTarget).closest(".pill")) {
+            // The searchbox loses focus while the search
+            // pill element gains focus.
+            // We do not consider the searchbox to actually
+            // lose focus when a pill inside it gets selected
+            // or deleted by a click.
+            return;
+        }
```

Comment:
Does this selector need to be more specific?  I worry about clicking on a pill in the compose box, or something, being misinterpreted